client/src/test/java/com/vesoft/nebula/encoder/TestEncoder.java

```diff
@@ -162,96 +167,106 @@ public int getPartId(int partNum, String vertexId) {
                                       Long ranking) {
         int edgeType = cacheImplTest.getEdge(spaceName, edgeName).edge_type;
         return codec.edgeKeyByDefaultVer(getSpaceVidLen(spaceName),
-            getPartId(getPartSize(spaceName), dstId),
-            dstId.getBytes(),
-            -edgeType,
-            ranking,
-            srcId.getBytes());
+                getPartId(getPartSize(spaceName), dstId),
+                dstId.getBytes(),
+                -edgeType,
+                ranking,
+                srcId.getBytes());
     }
 
     @Test()
     public void testVertexKey() {
         // test vertexKey
-        byte[] vertexKey = getVertexKey("test_space","Tom", "person");
+        byte[] vertexKey = getVertexKey("test_space", "Tom", "person");
         String hexStr = Hex.encodeHexString(vertexKey);
         String expectResult = "01d80100546f6d00000000000000000000000000"
-            + "0000000002000000";
+                + "0000000002000000";
+        Assert.assertArrayEquals(expectResult.getBytes(), hexStr.getBytes());
+
+        // test orphanVertexKey
+        byte[] orphanVertexKey = getOrphanVertexKey("test_space", "Tom", "person");
+        hexStr = Hex.encodeHexString(orphanVertexKey);
+        expectResult = "07d80100546f6d0000000000000000000000000000000000";
```

Comment:
prefer 
```
"07" + "d80100546f6d0000000000000000000000000000000000"

and vertexKey = "01" + "d80100546f6d0000000000000000000000000000000000" + "02000000"
```

Reply:
and "d80100546f6d0000000000000000000000000000000000" could use the same variable.